### PR TITLE
fix Tkinter checks for Python 3.x, change check to see whether Tkinter should be built

### PR DESCRIPTION
**Pull request by boegel**
follow up for #1156

cc @wpoely86

### CODE REVIEW

File: easybuild/easyblocks/p/python.py [hunk 1]
```diff
@@ -156,6 +150,13 @@ def sanity_check_step(self):
         """Custom sanity check for Python."""
 
         pyver = 'python' + '.'.join(self.version.split('.')[:2])
+        shlib_ext = get_shared_lib_ext()
+
+        if '--with-tcltk' in self.cfg['configopts']:
```
Comment (wpoely86) [inline easybuild/easyblocks/p/python.py]: why this change?

Comment (boegel) [inline easybuild/easyblocks/p/python.py]: Well, it's a better way to check if Tkinter should be there or not; it gives the flexibility to add `--with-tcltk` to `configopts` in the easyconfig itself without having `Tk` as a dependency, e.g. to use the system Tk.

Comment (wpoely86) [inline easybuild/easyblocks/p/python.py]: well, we don't usually count on any system dep

**Comment by boegel** [comment]
Thanks for the review @wpoely86, verified that this fixes the sanity check for Python v3.x, so going in.
